commit 78d4d2202a2861ef618f4e7f394e545a9a6d2339
Author: Ondřej Čertík <ondrej.certik@gmail.com>
Date:   Fri Mar 6 14:00:09 2015 -0700

    Fix include path
    
    The Python script requires access to exodusII.h, but it is importing it from
    $PREFIX/inc, instead of $PREFIX/include. This patch fixes it.

diff --git a/exodus/exodus.py.in b/exodus/exodus.py.in
index 3784ae9..b54938c 100644
--- a/exodus/exodus.py.in
+++ b/exodus/exodus.py.in
@@ -56,7 +56,7 @@ def getExodusVersion():
   Parse the exodusII.h header file and return the version number or 0 if not
   found.
   """
-  for line in open(getAccessPath() + '/inc/exodusII.h'):
+  for line in open(getAccessPath() + '/include/exodusII.h'):
     fields = line.split()
     if (len(fields) == 3 and
         fields[0] == '#define' and

